app/helpers/verification_helper.rb

```diff
@@ -293,7 +293,10 @@ def ridp_admin_actions(ridp_type, person)
   end
 
   def build_admin_actions_list(v_type, f_member)
-    if f_member.consumer_role.aasm_state == 'unverified' || VerificationType::ADMIN_CALL_HUB_VERIFICATION_TYPES.exclude?(v_type.type_name)
+    if EnrollRegistry.feature_enabled?(:ai_an_self_attestation) && v_type.type_name == VerificationType::AMERICAN_INDIAN_STATUS
+      attested_only = ["Verify", "Reject", "Call HUB", "Extend"]
```

Comment:
This seems like a good opportunity to break up the values in `::VlpDocument::ADMIN_VERIFICATION_ACTIONS` to each be defined as their own constant, and we can convert `ADMIN_VERIFICATION_ACTIONS` into a combination of those new constants.  If we do that, we can just return something like `::VlpDocument::ViewHistory` here. This way we avoid hardcoding values outside of a class where the definition of what those values are really belongs to that class.